apptools/logger/ring_buffer.py

```diff
@@ -44,10 +44,10 @@ def get(self):
 # sample of use
 """x=RingBuffer(5)
 x.append(1); x.append(2); x.append(3); x.append(4)
-print x.__class__,x.get()
+print(x.__class__,x.get())
```

Comment:
This is an example of something that's going to behave differently under Python 2, printing a tuple. But given that Python 2 is mostly dead and that this is commented-out code, it's hard to care. (If we do care, the right fix is to do `from __future__ import print_function` at the top of the module.)